proposals/ThirdPartyPackageManagers.md

```diff
@@ -32,3 +32,14 @@ The `name` attribute would specify the command-line package manager to be used,
 ## Framework Tag
 
 The `src` attribute can be a relative path or a URL. The `type` attribute specifies the type of framework that corresponds to the `engine` supported. In this case type `podspec` means that the lib should be installed using the `cocoapods` package manager, and `nuspec` for the `nuget` package manager.
+
+## Spec Argument
```

Comment:
I'm not necessarily against this but the spec attribute would not be used for Android. When specifying a gradle package in android the framework tag looks like:

```
<framework src="com.android.support:support-v13:23+" />
```

where the last bit `23+` indicates what version of the Support v13 library to use. In this case version 23 or up.

Would we include a spec attribute for Android so that the tag becomes:

```
<framework src="com.android.support:support-v13" spec="23+" />
```

Mind you we'll still have to support the old way for backwards compatibility.